src/sidebars/search/search-controller.js

```diff
@@ -21,7 +21,9 @@
         }
 
         select(stock) {
-            this.selectionService.select(stock);
+            if (stock.favourite) {
+                this.selectionService.select(stock);
+            }
```

Comment:
Not sure if I understand this. The selection can be changed from the compact view if the stock is a favourite, but why?